math/libfaiss/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        facebookresearch faiss 1.6.4 v
+github.setup        facebookresearch faiss 1.7.0 v
 name                libfaiss
-epoch               1
+epoch               0
```

Comment:
The epoch can never be reduced. Please set this back to 1.

Reply:
Whoops thanks that was supposed to be `revision`. Fixed.